### PR TITLE
Add cnfde6 cluster and site-specific configs to our demo cluster

### PR DESCRIPTION
**Pull request by lack**
Note: Do not merge yet; I will merge this as part of a demo recording

### CODE REVIEW

File: manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+apiVersion: policy.open-cluster-management.io/v1
```
Comment (yrobla) [inline manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml:1]: Ok, two things...
First, we need to discuss here if we want to deploy apps using policies. The idea that i had on mind, is to only use policies for the cluster configuration (network, settings, names...) . But for the specific workloads, i was thinking in just relying on common subscriptions. So you will have the pod definition in some folder for your site, and then you will have a subscription pointing to it. But no policies for that, as it's something at application level.
Second thing, i'd prefer if we split the PR in two. First to enroll/configure your cluster into acm, second one to create the application.
@yuvalk @vitus133 what are your thoughts about the way of configuring applications?

**Review by yrobla — CHANGES_REQUESTED** [review]

File: manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml (hunk 1 above)
Comment (lack) [reply manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml:1]: This branch is split into 3 commits:
- Add the spoke
- Add a site config
- Add the application
I'd be happy to split it into separate PRs if we'd like that better.

Maybe I'm not fully understanding the question about workloads and policies / subscriptions - Can you point me at an example of what you mean about the pod description and how I should do that without a policy?

Also for workloads, especially sriov workloads, we would need a consistent way of naming the SRIOV VFs in order to share the pod definitions from one place to another, or maybe use kustomization to alter a basic definition from site to site? Do we want to standardize something within our lab environment?  Right now I'm using 'nic{1,2}port{1,2}' in my SRIOV configs.  We could consider things like "fh{1,2}" "bh{1,2}"?  Or just leave that site-specific.

Comment (yrobla) [reply manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml:1]: Yep, i'd prefer if you can split into 3 prs, so we can review easily...
About the application, i mean that you could do it just with kustomization, not relying on policies. Just create a subscription but this should not point to a policy, but to the repo containing the real manifests. See for example what i do on https://github.com/redhat-nfvpe/flexran-manifests/blob/master/acm_subscription/sites/sample_site/sub_performance_profile.yaml ... it points to https://github.com/redhat-nfvpe/flexran-manifests/tree/master/cluster_configuration/sites/sample_site/performance_profile . That is just the plain manifests, not any policies. You could do the same with a subscription, pointing to the pod creation manifests, not to any policies that create them.

About the naming, i'll defer to @vitus133 ... he defined those objects and has the structure on mind... what do you think , Vitaly?

**Comment by vitus133** [comment]
Hi,
I added the network names to the resourceName because I thought that when
you define a pod, you probably want to know which network you are
connecting to (fronthaul, midhaule etc). Also, there is an option to define
several network node policies on the same NIC port with several different
resource names (for example allocate VF 0..3 to one resource and VF 4..7 to
another, from the same ens1f0 physical port).
Hence I think the best would be to have all of the important things (nic,
port, network name, driver type (vfio-pci or netdevice) to appear in the
resource name. I only concerned it will become too long for humans

File: manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml (hunk 1 above)
Comment (lack) [reply manifests/sites/cnfde6_site/10_testpmd_pod_policy.yaml:1]: If it's just for review purposes, you can click on the individual commits in the list above:

image

I'll look at your examples later today.

**Pull request closed without merging**